src/extractor/guidance/coordinate_extractor.cpp

```diff
@@ -32,7 +32,7 @@ const constexpr double LOOKAHEAD_DISTANCE_WITHOUT_LANES = 10.0;
 // smaller widths, ranging from 2.5 to 3.25 meters. As a compromise, we use
 // the 3.25 here for our angle calculations
 const constexpr double ASSUMED_LANE_WIDTH = 3.25;
-const constexpr double FAR_LOOKAHEAD_DISTANCE = 20.0;
+const constexpr double FAR_LOOKAHEAD_DISTANCE = 40.0;
```

Comment:
Coordinate extraction was broken. It stopped on the first segment that was longer than 20, not the total distance. Therefore I could previously reduce it without negative effects. Turned out to be a bug.